src/classes/ADDR_Addresses_TEST.cls

```diff
@@ -1183,6 +1183,121 @@ public with sharing class ADDR_Addresses_TEST {
 
     /*********************************************************************************************************
     @description
+        Add an existing Contact to a Household. 
+        Pre-populate the Contact's address with the Household Address
+        in the same way Manage Household pre-populates Contact's Mailing Address with Account's Billing Address.  
+    verify:
+        The Contact's current Address is equal to the new Household Address.
+        Contact's "Primary Address Type" is the same as the Address' "Address Type".
+    **********************************************************************************************************/        
+    static testMethod void addExistingContactToNewHouseholdShouldSyncPrimaryAddressTypeWithAddressType() {
+        if (strTestOnly != '*' && strTestOnly != 'addExistingContactToNewHouseholdShouldSyncPrimaryAddressTypeWithAddressType') return;
+        
+        UTIL_CustomSettingsFacade.getContactsSettingsForTests(
+            new npe01__Contacts_and_Orgs_Settings__c (
+                npe01__Account_Processor__c = CAO_Constants.HH_ACCOUNT_PROCESSOR,
+                Simple_Address_Change_Treated_as_Update__c = true
+            )
+        );
+
+        Integer hhSize = 1;
+        Integer contactSize = 1; 
+        createHHTestData(hhSize, contactSize);
+
+        Account account = new Account(Name = 'AccountWithoutAddress');
+        insert account;
+        Contact contact = new Contact(LastName = 'TestPrimaryAddressTypeExistingContact', AccountId = account.Id);
+        insert contact;
+
+        Test.startTest();
+
+        ADDR_Addresses_TDTM.hasRunAddrTrigger = false;
+
+        contact.accountId = listAccT[0].Id;
+        update contact = populateContactAddress(contact, listAddrT[0]);
```

Comment:
The "populateContactAddress" will populate Contact address from the source address and return the transformed Contact. The transformation of the Contact is returned even though the update of the address on Contact is by reference (the input argument Contact address update is visible after the method call). 
We could have "update populateContactAddress(contact, listAddrT[0]);" statement instead, but for the clarity and readability reasons, the method output is assigned to the "contact" variable. 
If you have other suggestions, please let me know.